core/src/test/scala/org/apache/spark/deploy/SparkSubmitUtilsSuite.scala

```diff
@@ -79,7 +79,7 @@ class SparkSubmitUtilsSuite extends SparkFunSuite with BeforeAndAfterAll {
 
   test("create additional resolvers") {
     val repos = "a/1,b/2,c/3"
-    val settings = SparkSubmitUtils.buildIvySettings(Option(repos), None)
+    val settings = SparkSubmitUtils.buildIvySettings(Option(repos), Some(tempIvyPath))
```

Comment:
BTW, is this required to change? I think usual case in production is to work without specifying the temp dir.

Reply:
Yes, this is only required for tests and if you check the test `SPARK-10878: test resolution files cleaned after resolving artifact` at the end, it checks whether the resolution files are cleaned at the end or not, but that use `tempIvyPath` whereas when we don't pass it here what happens is it picks user's .ivy location. This change is slightly separate as its just standardizes using `tempIvyPath` across the tests in this suite.